viam_cartographer.go

```diff
@@ -415,18 +454,18 @@ func initCartoFacade(ctx context.Context, cartoSvc *CartographerService) error {
 
 	cartoCfg := cartofacade.CartoConfig{
 		Camera:             cartoSvc.lidar.Name(),
-		MovementSensor:     cartoSvc.imu.Name(),
+		MovementSensor:     cartoSvc.movementSensorName,
 		ComponentReference: cartoSvc.lidar.Name(),
 		LidarConfig:        cartofacade.TwoD,
 		EnableMapping:      cartoSvc.enableMapping,
 		ExistingMap:        cartoSvc.existingMap,
 	}
 
-	if cartoCfg.MovementSensor != "" {
+	if cartoSvc.imu != nil {
 		cartoSvc.logger.Warn("IMU configured, setting use_imu_data to true")
 		cartoAlgoConfig.UseIMUData = true
 	} else {
-		cartoSvc.logger.Debug("No IMU configured, setting use_imu_data to false")
+		cartoSvc.logger.Warn("No IMU configured, setting use_imu_data to false")
```

Comment:
I believe this was intentionally a Debug log, since a user might interpret the lack of IMU as a problem

Reply:
Hmn that's a good point. My thinking here was that up until now when a user tried to add a movement sensor that could not be used as an IMU, we would throw an error during `ValidateGetIMUData` and SLAM would not start running. Now, a user can add a movement sensor that does not support an IMU (and possibly not an odometer either) and SLAM will still start up but simply not add any IMU/odometer data. 

In this case I thought it might be useful to inform the user that their movement sensor can not be used as an IMU nor odometer.

I wonder if this is a better solution: We could make it a warning when a movement sensor was added but can't be used as an IMU/odometer. When no movement sensor was added, we'll leave it as a debug message.

What do you think? I'm not really tied to any of these solutions and am happy to just make this a debug message again.

Reply:
> I wonder if this is a better solution: We could make it a warning when a movement sensor was added but can't be used as an IMU/odometer. When no movement sensor was added, we'll leave it as a debug message.

I think this is a good solution!